### PR TITLE
Feature ajs add campuses to send connection report job

### PR DESCRIPTION
**Pull request by ajs1037**
Added Chandler, North Phoenix and Verrado to the send connection report job

### CODE REVIEW

File: church.ccv.Utility/Jobs/SendConnectionReport.cs [hunk 1]
```diff
@@ -302,6 +308,81 @@ public void Execute( IJobExecutionContext context )
                 }
             }
 
+            // send Chandler email
+            if (chandlerEmailList.Any())
+            {
+                var mergeFields = new Dictionary<string, object>();
+                mergeFields.Add("Title", "Surprise Campus");
```
Comment (jhawkzz) [inline church.ccv.Utility/Jobs/SendConnectionReport.cs]: This needs to match the campus name

File: church.ccv.Utility/Jobs/SendConnectionReport.cs [hunk 2]
```diff
@@ -452,6 +533,81 @@ public void Execute( IJobExecutionContext context )
                 }
             }
 
+            // send North Phoenix email
+            if (northphoenixEmailList.Any())
+            {
+                var mergeFields = new Dictionary<string, object>();
+                mergeFields.Add("Title", "Surprise Campus");
```
Comment (jhawkzz) [inline church.ccv.Utility/Jobs/SendConnectionReport.cs]: This needs to match the campus name

File: church.ccv.Utility/Jobs/SendConnectionReport.cs [hunk 3]
```diff
@@ -677,6 +833,81 @@ public void Execute( IJobExecutionContext context )
                 }
             }
 
+            // send Verrado email
+            if (verradoEmailList.Any())
+            {
+                var mergeFields = new Dictionary<string, object>();
+                mergeFields.Add("Title", "Surprise Campus");
```
Comment (jhawkzz) [inline church.ccv.Utility/Jobs/SendConnectionReport.cs]: Campus name

**Comment by ajs1037** [comment]
@jhawkzz I moved the function outside of the execute. You told me to merge as soon as I did that, but I would like you to check my code one last time. Moving the function outside of the execute function wasn't too straightforward and I just want to make sure I did everything correctly. I complied and made sure everything ran. No errors. I'm sure!! Thank you so much for your help on this. 


Here is a summary of what I did:


1. Moved SendCampusEmailList out of the Execute function
2. Had to add a rockContext parameter to SendCampusEmailList
3. Added the rockContext parameter to every SendCampusEmailList function call inside execute.

Current errors at this point:
    Severity	Code	Description	Project	File	Line	Suppression State
    Error	CS0103	The name 'sevenDaysAgo' does not exist in the current context	
    Error	CS0103	The name 'sevenDaysAgo' does not exist in the current context	
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'thirtyDaysAgo' does not exist in the current context
    Error	CS0103	The name 'connectionPageId' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'systemEmail' does not exist in the current context
    Error	CS0103	The name 'context' does not exist in the current context

4. Moved time frames and get system email code from Execute to SendCampusEmailList
        // timeframes
        var sevenDaysAgo = RockDateTime.Now.AddDays( -7 );
        var thirtyDaysAgo = RockDateTime.Now.AddDays( -30 );

        // get system email
        SystemEmail systemEmail = null;
        if ( systemEmailGuid.HasValue )
        {
            SystemEmailService emailService = new SystemEmailService( rockContext );
            systemEmail = emailService.Get( systemEmailGuid.Value );
        }

5. Had to add systemEmailGuid and connectionPageId as parameters to the SendCampusEmailList funtion.
6. Moved  context.Result = "Emails sent.";

**Pull request merged**